src/chainstate/stacks/mod.rs

```diff
@@ -228,6 +229,9 @@ impl fmt::Display for Error {
             Error::MemPoolError(ref s) => fmt::Display::fmt(s, f),
             Error::NoTransactionsToMine => write!(f, "No transactions to mine"),
             Error::PoxAlreadyLocked => write!(f, "Account has already locked STX for PoX"),
+            Error::PoxExtendNotLocked => {
+                write!(f, "Account has not already locked STX for PoX extend")
```

Comment:
newbie question: is "stacking" == "locking"?

Reply:
yes, in this context